CMakeLists.txt

```diff
@@ -13,7 +13,10 @@ set(CMAKE_USER_MAKE_RULES_OVERRIDE
 
 project(Celeritas VERSION "${Celeritas_VERSION}" LANGUAGES CXX)
 
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake")
+list(APPEND CMAKE_MODULE_PATH
+  "${CMAKE_CURRENT_LIST_DIR}/cmake"
+  "${CMAKE_CURRENT_LIST_DIR}/cmake/external"
```

Comment:
I don't think we should add this subdirectory to the the CMAKE_MODULE_PATH, I really meant for the file to only be loaded explicitly.

Reply:
We have to either do this or escape out the CudaRdcUtils include when vecgeom is disabled.

Reply:
The former is 'deadly' (i.e. it is equivalent to only ever use the version in Celeritas no matter what).
The later is trivial (add `OPTIONAL` to the `include` call).

Reply:
But `OPTIONAL` does not seem to work as [advertised](https://cmake.org/cmake/help/latest/command/include.html#include) out of the box.  I am investigating.

Reply:
Oh.  That's because there is more than one such include :(